OpenSim/Actuators/Millard2012EquilibriumMuscle.cpp

```diff
@@ -1601,59 +1595,41 @@ Millard2012EquilibriumMuscle::estimateMuscleFiberState(
         iter++;
     }
 
-    // Populate the result map.
-    ValuesFromEstimateMuscleFiberState resultValues;
+    MuscleStateEstimate result;
+    result.solutionError = ferr;
 
-    if(abs(ferr) < aSolTolerance) {  // The solution converged.
+    if(abs(ferr) < aSolTolerance) {
```

Comment:
Re-add `  // the solution converged`: it's a useful comment when scan-reading the code